lib/ui/pages/coursedetail/screen/album_page.dart

```diff
@@ -0,0 +1,115 @@
+import 'package:flutter/material.dart';
+import 'package:flutter_app/src/store/picture_storage.dart';
+import 'package:photo_view/photo_view.dart';
+import 'dart:io';
+
+class AlbumPage extends StatefulWidget {
+  final String courseId;
+
+  const AlbumPage({Key? key, required this.courseId}) : super(key: key);
+
+  @override
+  State<AlbumPage> createState() => _AlbumPageState();
+}
+
+class _AlbumPageState extends State<AlbumPage> {
+  bool isLoading = true;
+
+  late int pictureNum;
+  late List<Picture> pictures = [];
+
+  dynamic picturesInfo;
```

Comment:
Declare this inside useCourseIdToGetPicturePaths(), it does not have other usages.